code/src/components/Colour.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const Colour = ({ color, setColor }) => {
+  const handleColorChange = (event) => {
+    setColor(event.target.value);
+  };
+
+  return (
+    <form required onSubmit={(event) => event.preventDefault()}>
+      <label htmlFor="colorPicker">
```

Comment:
The colorpicker - cool function! Will try to remember for future fun projects :)